core/services/workflows/syncer/workflow_registry.go

```diff
@@ -137,61 +187,116 @@ func WithReader(reader types.ContractReader) func(*workflowRegistry) {
 	}
 }
 
+// TODO Beholder implementation of this
+type workflowRegistryStats interface {
```

Comment:
Can we file a ticket to track this TODO and put it in Street's epic?

Reply:
(Or better yet: can we just implement today? It feels like it should be just a couple of extra lines).